stl/inc/format

```diff
@@ -2545,7 +2549,7 @@ _NODISCARD _OutputIt _Write_integral(
     _OutputIt _Out, const _Integral _Value, _Basic_format_specs<_CharT> _Specs, _Lazy_locale _Locale) {
     if (_Specs._Type == 'c') {
         if (!_In_bounds<_CharT>(_Value)) {
-            _THROW(format_error("integral cannot be stored in charT"));
+            _Throw_format_error("integral cannot be stored in charT");

```

Comment:
Pre-existing: should this use `char/wchar_t` instead of `charT` to be consistent with 1673?

Reply:
That seems like an improvement, slightly less likely to confuse the programmer. This set of changes is focused enough that a few drive-by cleanups won't be too distracting.

Reply:
> This set of changes is focused enough that a few drive-by cleanups won't be too distracting.

I felt bad suggesting even this, but since this is technically a series of PRs filled with drive-bys, what's one more?